js/sykepengesoknad/data/soknader/soknaderSagas.js

```diff
@@ -222,7 +222,7 @@ function* watchOpprettSoknadUtland() {
 }
 
 function* watchEndringSoknad() {
-    yield takeEvery(SOKNAD_ENDRET, oppdaterSporsmal);
+    yield takeLatest(SOKNAD_ENDRET, oppdaterSporsmal);
```

Comment:
Da forkaster vi sekvensielle endringer om vi har en senere versjon av søknaden. Kan vi få noen artige out of order problemer med dette? Jeg ser for meg at det er mulig at bruker ender opp med et spørsmålsett som er forskjellig fra det som er koblet til søknaden i syfosoknad, men vil da ikke ha mulighet til å sende inn søknaden..

Reply:
Det stemmer, dette funket ikke helt som ønsket i Q1.